front/src/routes/map/Map.jsx

```diff
@@ -1,11 +1,10 @@
 import { Component } from 'preact';
-import leaflet from 'leaflet';
 import { Text } from 'preact-i18n';
 import { connect } from 'unistore/preact';
+import { route } from 'preact-router';
+import leaflet from 'leaflet';
 
-import 'leaflet/dist/leaflet.css';
```

Comment:
Why do you remove this?

Reply:
Same as previous i think => I will correct